src/client/containers/ProfilePage/BoardBanners.component.js

```diff
@@ -0,0 +1,55 @@
+import React, { useContext } from 'react';
+
+import ProfileProps from './ProfileContext';
+import BoardCard from '../../components/BoardCard/BoardCard.component';
+import DropDownMenuMyBoards, {
+  DropDownMenuJoinedBoards,
+} from './BoardCardDropdown.component';
+import NewBoard from './NewBoard.component';
+import './BoardBanners.styles.css';
+
+export default function BoardBanners() {
+  const { onMyBoards, joinedBoards, myBoards } = useContext(ProfileProps);
+
+  if (!joinedBoards) {
+    return null;
+  }
+  return (
+    <div className="existing-boards">
+      {onMyBoards && (
+        <div>
+          <NewBoard />
+          <div>
```

Comment:
you will have the put the board card (line 23 to line 29 ) inside of a link  <Link to={......./{board.id}>. Now when you click the board. It will open that particular board